native/swift/Example/Example/ListViewModel.swift

```diff
@@ -60,3 +97,34 @@ struct MyError: LocalizedError {
         underlyingError.localizedDescription
     }
 }
+
+struct ListViewSequence: AsyncSequence {
```

Comment:
Is this struct implementation equavilant of `asyncSequence.compactMap({ ($0 as? [ListViewDataConvertable]).asListViewData() })`?